sympy/integrals/meijerint_doc.py

```diff
@@ -12,7 +12,7 @@
 
 doc = ""
 
-for about, category in sorted(t.items()):
+for about, category in sorted(t.items(), key=lambda x: sorted(x[0], key=str)):
```

Comment:
From what I can tell, this has replaced deprecated sorting of types with sorting based on the unspecified order that `str` displays `frozenset`? Seems like there's little point sorting at all...

Reply:
This is just the documentation. We should probably change this to sort the frozenset. 

From what I understood, the sorting is needed (I made this commit originally a while ago).

Reply:
Why not just leave the frozenset as the tuple it was originally? Then this line will still be deterministic, but will no longer rely on type ordering.